js/navbar.js

```diff
@@ -11,8 +11,6 @@ window.addEventListener('DOMContentLoaded', (event) => {
       : 'c4a84431feaf604e89d1';
   userLoginEl.href = `https://github.com/login/oauth/authorize?client_id=${clientId}&state=${checkURL}`;
 });
-const hamburger = document.querySelector('.hamburger');
-const navMenu = document.querySelector('.nav-menu');
```

Comment:
Why this is removed ? Can we please check the mobile view if it's working fine or not ?
Also if possible you can share the recordings for it

Reply:
The variables **hamburger** & **navMenu** were already on the top that too using `const`. It gave me error when I started working on this issue, so I found them redundant and removed.
Sharing a Screenshot for reference:
![image](https://github.com/Real-Dev-Squad/website-welcome/assets/69259490/35c18854-2193-4ae6-9bbd-5a28398f0120)

Reply:
Here's the recording for Mobile view: 

https://github.com/Real-Dev-Squad/website-welcome/assets/69259490/bf6725da-64cf-41a1-9385-e837e76f7ef1

Reply:
The toggling is working fine 🙂

https://github.com/Real-Dev-Squad/website-welcome/assets/69259490/91ffb16e-5f26-4b5a-a039-5230236f171d